library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -584,6 +636,54 @@ public void notifyItemChangedInSection(Section section, int position) {
         callSuperNotifyItemChanged(getPositionInAdapter(section, position));
     }
 
+    /**
+     * Helper method that calculates the relative header position in the adapter and calls
+     * {@link #notifyItemChanged}. Do nothing if the section doesn't have header
```

Comment:
"do nothing" comments here and below are outdated, I will remove them in develop after merge